.changeset/dirty-ravens-judge.md

```diff
@@ -0,0 +1,43 @@
+---
+
+Time zone objects generated by `@commercetools-frontend/l10n` have been updated to use custom translated strings as labels.
+
+Not all time zones that were previously in the time zone objects have been assigned translations. Translations have been added in `packages/l10n/data/time-zones/`. There is a `core.json` and a file for each supported locale (`<locale>.json`). Transifex integration for these translations will be added in a subsequent PR.
+
+For those time zones that have not been translated, each have been mapped to a time zone that does have a translation in `packages/l10n/data/time-zones/translations-map.json`. This ensures that each IANA time zone identifier returned by `moment.tz.names()` will show an accurate translation.
+
+The IANA [peridically updates](https://www.rfc-editor.org/rfc/rfc6557.html) its' [list of timezones](https://www.iana.org/time-zones). When the IANA releases a new set of time zones, [it will be added](https://momentjs.com/docs/#/-project-status/) to `moment-timezone`. In order to reduce the overall need for maintainance, the `l10n/scripts/generate-10n-data.js` script has been updated, and `l10n/scripts/parse-unhandled-time-zones.js` has been added to handle CLI input.
+
+When `generate-l10n-data` is run, it gets the list of all time zones being returned by `moment-timezone` by running `moment.tz.names()`, and compares it to the list of translated (included) and mapped (excluded) time zones in `core.json` and `translations-map.json`.
+
+For each unhandled time zone returned, CLI will prompt the user to either `translate`, `exclude`, or `ignore` it.
+
+If the user selects `translate`, they must enter an english language translation string. That string will then be added to `core.json` and each `<locale>.json` for subsequent translation. It will also be added as a key in `translations-map.json` so that subsequent time zones can be mapped to its' translation if they are excluded.
+
+If the user selects `exclude`, they must enter the IANA time zone id of a time zone that currently has a translation in order to map the time zone to a translation string. The `excluded` translation is then added to the array for the give translated time zone in `translations-map.json`, e.g.:
+`"America/New_York": ["EST", "EDT"]`
+
+If the user selects `ignore`, the unhandled time zone will be ignored until the `generate-l10n-data` script is run again.
+
+This results in time zones being exported from the `l10n` package with the following shape:
+
+```js
+{
+    [timeZoneId]:{
+        id: <time zone  id>,
+        abbr: <time zone abbreviation>,
+        label: <translated label from locale.json>,
+        offset: <GMT offset>,
+        translationFor: [<array of time zone ids that fall back to this translation>]
+    },
+    'America/New_York': {
+        id: 'America/New_York',
+        abbr: 'EST',
+        label: 'Eastern Time - New York',
+        offset: '-05:00',
+        translationFor: ['EST', 'EDT']
+    }
+}
+```
```

Comment:
Thanks for the detailed explanation.

For a changeset though, this information is not really relevant and is more for us maintainers to know.

Can you move this to the README of the package maybe? There is a section about "Generating localization data".

In the changeset I would simply say the following:

```suggestion
The localized data has been updated. In particular, the time zones data has been updated and restructured to be more usable and readable.
```

Reply:
changeset and readme have been updated